src/main/java/reposense/git/GitUtil.java

```diff
@@ -77,11 +77,11 @@ static String convertToFilterAuthorArgs(Author author) {
     /**
      * Returns the {@code String} command to specify the file formats to analyze for `git` commands.
      */
-    static String convertToGitFormatsArgs(List<Format> formats) {
+    static String convertToGitFormatsArgs(List<FileType> formats) {
```

Comment:
I personally advise against renaming the arg name to `fileTypes` as there's `groups` in `FileTypeManager` to prevent any confusion that this method may include groups as well.